CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,485 @@
+        }
+      },
+      "required": [
+          "reportURLs",
+          "reportHash"
+      ]
+    },
+    "summary": {
+      "type": "string",
+      "description": "A string of the summary of the certification/audit. "
+    },
+    "disclaimer": {
+      "type": "string",
+      "description": "A string of the legal disclaimer from the Certificate Issuer"
+    },
+    "scripts": {
+      "type": "array",
+      "items": [
+        {
+            "smartContractInfo": {
+                "type": "object",
+                "properties": {
+                    "era": {
+                        "type": "string",
+                        "description": "Describe which era this script was written for."
+
+                    },
+                    "compiler": {
+                        "type": "string",
+                        "description": "Compiler used for this script."
+                    },
+                    "compilerVersion": {
+                        "type": "string",
+                        "description": "Compiler version used for this script."
+                    },
+                    "optimizer": {
+                        "type": "string",
+                        "description": "Post compilation optimizer used for this script."
+                    },
+                    "optimizerVersion": {
+                        "type": "string",
+                        "description": "Post compilation optimizer version used for this script."
+                    },
+                    "progLang": {
+                        "type": "string",
+                        "description": "Programming language used for this script"
+                    },
+                    "repository": {
+                        "type": "string",
+                        "description": "URL to the repository of the script"
+                    }
+                }
+            },
+            "fullScriptHash": {
+              "type": "string",
+              "description": "Prefix and script hash or script hash+staking key"
+            },
+            "scriptHash": {
+              "type": "string",
+              "description": "Script hash or script hash+staking key"
+            },
+            "contractAddress": {
+              "type": "string",
+              "description": "Script on-chain address"
+            },
+            "required": [
+                "scriptHash",
+                "contractAddress"
+              ]
+        }
+      ]
+    }
+  },
+
+  "required": [
+    "subject",
+    "schemaVersion",
+    "certificationLevel",
+    "certificateIssuer",
+    "report",
+    "summary",
+    "disclaimer",
+    "scripts"
+  ]
+}
+```
+
+### Example
+
+```json
+{
+  "subject": "d684512ccb313191dd08563fd8d737312f7f104a70d9c72018f6b0621ea738c5b8213c8365b980f2d8c48d5fbb2ec3ce642725a20351dbff9861ce9695ac5db8",
+  "schemaVersion": 1,
+  "certificationLevel": 1,
+  "certificateIssuer": {
```

Comment:
Maybe we should have separate CIP for certificate issuers, the reason being is that imagine some of this data changes, the consequence would be that all certificates would have to be resubitted.

Reply:
@matiwinnetou cross-referencing https://github.com/cardano-foundation/CIPs/pull/499#discussion_r1160800134 (@Ryun1 @MIxAxIM)

Reply:
resolving as per https://github.com/cardano-foundation/CIPs/pull/499#discussion_r1224274902 (i.e. this is a duplicate)